packages/js/src/ui/config/default-localization.ts

```diff
@@ -0,0 +1,10 @@
+import { Localization } from '../context/LocalizationContext';
+
+export const defaultLocalization: Localization = {
+  inbox: {
+    title: 'Inbox',
+  },
+  notification: {
+    mainButtonText: 'Accept',
```

Comment:
Is it main button or primary button? Please ensure parity in terminology with appearance prop.

Reply:
this is kind of a placeholder :P I'll remove it completely.